### DISCUSSION TITLE
Xiaomi scale 1st gen

### DISCUSSION THREAD

**Discussion by jjusko** [custom-components/ble_monitor#259] [General]
Is it possible to add support for xiaomi scale 1st gen.?
Thanks

**Comment by Ernst79** [comment]
Possibly, although we did not find useful messages in the 2nd generation scale. To find out  if it is possible, we need some advertisements, see for instructions on how to get these the explanation in the FAQ

https://github.com/custom-components/ble_monitor/blob/master/faq.md#my-sensor-from-the-xiaomi-ecosystem-is-not-in-the-list-of-supported-ones-how-to-request-implementation

**Comment by Ernst79** [comment]
Support is added for Xiaomi Scales, both gen 1 and 2.
